### ISSUE TITLE
prettier-atom failure. can't find prettier plugins

### ISSUE THREAD

**Issue by shellscape** [prettier/prettier-atom#682]
I'm wresting with the following error message:

```
prettier-atom failed: Cannot find module 'prettier-plugin-package' from '/'
```

We've a (unfortunately private) repo that's setup with ESLint and defines the Prettier config within the ESLint config. Please see below for the full configuration:

<details>
module.exports = {
  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
  extends: [
    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
  ],
  parserOptions: {
    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
    sourceType: 'module', // Allows for the use of imports
  },
  plugins: ['prettier'],
  rules: {
    '@typescript-eslint/explicit-function-return-type': 'off',
    '@typescript-eslint/explicit-member-accessibility': 'off',
    'no-unused-vars': ['error', { args: 'after-used' }],
    'prettier/prettier': [
      'error',
      {
        arrowParens: 'always',
        plugins: ['prettier-plugin-package'],
        printWidth: 120,
        singleQuote: true,
        semi: true,
        tabWidth: 2,
        trailingComma: 'all',
      },
    ],
  },
};
</details>

This config works pretty well when running `npx eslint`. Likewise, running `prettier --write --plugin=prettier-plugin-package` from anywhere in the repo works without issue, and we've confirmed that the plugin in question exists in `node_modules` and works correctly. However, when using `prettier-atom` with this setup, attempting to save a `package.json` file results in the error message pasted above. `prettier-atom` seems to be struggling with loading plugins, and I'm not sure why. If I had to guess, it may be telling prettier the wrong location from which to load plugins (perhaps the atom plugin directory?). I can find no logs specific to prettier-atom or those which contain information about prettier failures.

<details>
Atom version: 1.44.0
prettier-atom version: 0.58.2
prettier: bundled
prettier version: 1.18.2
prettier-eslint version: 9.0.1
prettier-atom configuration: {
  "formatOnSaveOptions": {
    "enabled": true,
    "whitelistedGlobs": [
      "package.json"
    ],
    "respectEslintignore": true,
    "showInStatusBar": false,
    "excludedGlobs": [],
    "isDisabledIfNotInPackageJson": false,
    "isDisabledIfNoConfigFile": false,
    "ignoreNodeModules": true
  },
  "useEslint": true,
  "useStylelint": false,
  "useEditorConfig": true,
  "prettierEslintOptions": {
    "prettierLast": false
  }
}
</details>

**Comment by shellscape** [comment]
I've narrowed this down to _how_ `prettier-atom` is executing `prettier`, but not exactly _where_ in code this is happening. Apparently, within `prettier` the value of `process.cwd()` is returning `/`. Hence the odd error message that it can't find the plugin at `/`.  (I sussed this out by editing prettier's code directly to also provide the cwd in the error message, and my editing the extension's code to also provide the stack in the error message)
